addons/mrp/models/product.py

```diff
@@ -64,6 +64,18 @@ def _compute_used_in_bom_count(self):
         for product in self:
             product.used_in_bom_count = self.env['mrp.bom'].search_count([('bom_line_ids.product_id', '=', product.id)])
 
+    def get_components(self):
+            """ Return the components list ids for kit product.
+             Return the product itself otherwise"""
+            self.ensure_one()
```

Comment:
double indent
docstring not clear
get_components doesn't seem appropriate (most of the times there won't be any notions of components)